src/algebra/group_power/basic.lean

```diff
@@ -111,8 +111,10 @@ by rw [← nsmul_eq_smul, nat.add_comm, nsmul_add', nsmul_one', nsmul_eq_smul]
 theorem pow_two (a : M) : a^2 = a * a :=
```

Comment:
```suggestion
/-- Note that most of the lemmas about powers of two refer to it as `sq`. -/
theorem pow_two (a : M) : a^2 = a * a :=
```

Reply:
So that people thinking "why are there so few lemmas about `pow_two`" when using autocomplete have a breadcrumb to follow.